<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>313</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>312</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>313</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ToDoList</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #045757;
color: white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_8">
    <item row="6" column="2" colspan="2">
     <widget class="QLineEdit" name="new_task_title">
      <property name="minimumSize">
       <size>
        <width>93</width>
        <height>0</height>
       </size>
      </property>
      <property name="placeholderText">
       <string>عنوان...</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="4">
     <widget class="Line" name="line">
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="4">
     <widget class="QTextEdit" name="new_task_descryption">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>75</height>
       </size>
      </property>
      <property name="placeholderText">
       <string>descryption...</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="4">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="new_task_btn">
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: orange;
color: white;
border-radius: 20px;</string>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0" rowspan="3" colspan="4">
     <layout class="QGridLayout" name="tasks_gl"/>
    </item>
    <item row="6" column="0">
     <widget class="QDateTimeEdit" name="dateTimeEdit">
      <property name="minimumSize">
       <size>
        <width>130</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QComboBox" name="select">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>0</height>
       </size>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="placeholderText">
       <string>اولویت</string>
      </property>
      <item>
       <property name="text">
        <string>عادی</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>متوسط</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>بحرانی</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>313</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
